dao/request_params.go

```diff
@@ -14,12 +13,12 @@ type RequestParams struct {
 }
 
 func NewRequestParamsFromContext(c echo.Context) (*RequestParams, error) {
-	tenantId, err := util.GetTenantFromEchoContext(c)
+	tenantId, err := sourcesEcho.GetTenantFromEchoContext(c)
```

Comment:
What would you think of renaming this import `echoUtils`? Since it's the echo-related util functions just in their own package.